src/common/opencl.h

```diff
@@ -139,7 +139,10 @@ typedef struct dt_opencl_device_t
   int pinned_memory;
   // in OpenCL processing round width/height of global work groups to a multiple of this value.
   // reasonable values are powers of 2. this parameter can have high impact on OpenCL performance.
-  int clroundup;  
+  int clroundup;
+
+  // A bitfield that identifies the type of OpenCL device
+  unsigned int cltype;
```

Comment:
declared/added here.

but never used. can you clarify?

Reply:
I want/need this for following pr requiering this info to disable "tuning cl device" if onboard graphics.
Either i remove this here and add to following pr or keep it as is.
As you want ...

Reply:
No that's fine, just wanted to be sure it was not a missing commit for example. Thanks for the clarification.